Godeps/Godeps.json

```diff
@@ -121,23 +121,23 @@
 		},
 		{
 			"ImportPath": "github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud",
-			"Comment": "1.2.0",
-			"Rev": "773c7a284e1399960745d0e2c501f577a20d4947"
+			"Comment": "1.2.0-1-gf8e99590510076",
```

Comment:
Is it possible to vendor at a real tag rather than a "dirty" point?  I'd like to make that the normal thing, and have arbitrary SHAs be the rare case...

Reply:
Added a 1.3.0 tag